components/resources/ai_chat_ui_strings.grdp

```diff
@@ -28,7 +28,10 @@
     Ask Brave AI
   </message>
   <message name="IDS_CHAT_UI_API_ERROR" desc="Describes an error occured submitting the API request">
-    There was an error processing your message, please try again.
+    There was a network issue connecting to Leo, check your connection and try again
+  </message>
+  <message name="IDS_CHAT_UI_API_TOO_MANY_REQUESTS_ERROR" desc="Describes an error occured when max count of API requests exceeded">
+    You have reached the current rate limit for Leo. Please try again later
```

Comment:
@rmcfadden3, is this too technical?

Reply:
@mkarolin — oof. sorry for missing your ping.

Yeah, this error message is too technical. I checked with @rebron, and it sounds like rate limiting here basically means the number of questions you've asked in a given time span (i.e one day). So, given that, I'd suggest editing to something like:

`You've reached the maximum number of questions for Leo. Please try again tomorrow.`

… or something along those lines.

Oh, and I assume the rate limit is increased (or even non-existent) for Leo Premium? If yes, we could amend the second sentence to something like `Please try again tomorrow, or upgrade to Leo Premium.` (if the UI allows for that many characters).

Reply:
fixed: https://github.com/brave/brave-core/pull/19318/commits/458ba9c9e82f11cec4bf347efc6101f0c04df9f7#diff-dbd0ebfba47f26a3cbccd8f40b0660933c0040d1cbea77d2ac4ffe4e2d56f853R34

Reply:
cc:@nullhook Let's remove the ", or upgrade to Leo Premium." line.